lib/Wikia/src/Service/Helios/HeliosClientImpl.php

```diff
@@ -4,18 +4,23 @@
 use Wikia\Tracer\WikiaTracer;
 use Wikia\Util\GlobalStateWrapper;
 use Wikia\Service\Constants;
+use \Wikia\Util\Assert;
 
 /**
  * @Injectable(lazy=true)
  * A client for Wikia authentication service.
  *
  * This is a naive implementation.
  */
-class HeliosClientImpl implements HeliosClient
-{
-	const BASE_URI = "helios_base_uri";
-	const SCHWARTZ_TOKEN = "schwartz_token";
+class HeliosClientImpl implements HeliosClient {
+	const BASE_URI = 'helios_base_uri';
+	const SCHWARTZ_TOKEN = 'schwartz_token';
 	const SCHWARTZ_HEADER_NAME = 'THE-SCHWARTZ';
+	const USERNAME = 'username';
+	const PASSWORD = 'password';
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this hard-coded password. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS2068)